app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::UsersController < ApplicationController
+  include ApplicationHelper
+  before_action :authorize_user, only: [:destroy, :index]
```

Comment:
just do `before_action :authorize_user`. It will default to all actions in this controller.